src/components/MainPanel/index.js

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import { Route, Switch, Redirect, withRouter } from "react-router-dom";
+import FavoritesPanel from './FavoritesPanel';
+import ExplorePanel from './ExplorePanel';
+
+import { bindActionCreators } from "redux";
+import actions from "../../actions";
+import { connect } from "react-redux";
+
+const { loadSavedFavorites } = actions.creators;
+
+class MainPanel extends Component {
+  componentDidMount() {
+    this.props.loadSavedFavorites();
+  }
+
+  render() {
+    return (
+      <Switch>
+        <Route path="/explore" component={ExplorePanel} />
+        <Route path="/favorites" component={FavoritesPanel} />
+        <Redirect to="/explore" />
```

Comment:
It is a nice feature to send an object to handle the redirect and let the user know what happened before the redirect.